### DISCUSSION TITLE
Need option to use Rust compiler and ignore the .babel file

### DISCUSSION THREAD

**Discussion by timothyarmes** [vercel/next.js#30659] [Ideas]
### Describe the feature you'd like to request

Hi,

After upgrading to v12 and deleting my .babel file, everything works perfectly...

...except all my unit tests, which fail since jest needs the .babel file.

### Describe the solution you'd like

Can we have a way to force an opt-in to the Rust compiler regardless of the existence of the .babel file?

### Describe alternatives you've considered

My alternative is to keep using babel

**Comment by zetoke** [comment]
You can migrate to use https://github.com/swc-project/jest

It works great for me

**Comment by nunesmatheus** [comment]
Just faced the same problem. Replacing
``` js
transform: {
  "\\.[jt]sx?$": "babel-jest"
}
```
with
``` js
transform: {
  "\\.[jt]sx?$": ["babel-jest", { presets: ["next/babel"] }]
}
```
on **jest.config.js** fixed it for me!

**Comment by sannajammeh** [comment]
I agree, we currently use a custom babel config to transform imports for tree shaking. We have no need for this in development, an option to prefer SWC in next.config.js would be great.
